src/main/java/org/smartregister/domain/SubFormDefinition.java

```diff
@@ -0,0 +1,64 @@
+package org.smartregister.domain;
+
+import java.util.List;
+
+public class SubFormDefinition {
```

Comment:
This class should remain in core its not being used on this library

Reply:
Moved back to core project